dotCMS/src/main/java/com/dotcms/publisher/util/DependencyManager.java

```diff
@@ -244,29 +244,50 @@ public void setDependencies() throws DotSecurityException, DotDataException, Dot
 				} catch (DotSecurityException e) {
 					Logger.error(getClass(), "Couldn't add the Container to the Bundle. Bundle ID: " + config.getId() + ", Container ID: " + asset.getAsset(), e);
 				}
-			} else if(asset.getType().equals(PusheableAsset.FOLDER.getType())) {
+			} else if (asset.getType().equals(PusheableAsset.FOLDER.getType())) {
 				try {
-					Folder f = APILocator.getFolderAPI().find(asset.getAsset(), user, false);
-
-					if(f == null){
-						Logger.warn(getClass(), "Folder id: "+ (asset.getAsset() != null ? asset.getAsset() : "N/A") +" does NOT have working or live version, not Pushed");
+					final Folder f = APILocator.getFolderAPI().find(asset.getAsset(), user, false);
+
+					if(f == null) {
+						Logger.warn(
+								getClass(),
+								String.format(
+										"Folder id: %s does NOT have working or live version, not Pushed",
+										asset.getAsset() != null ? asset.getAsset() : "N/A"));
 					} else {
-						folders.add(asset.getAsset(), f.getModDate());
-						foldersSet.add(asset.getAsset());
+						f.ifSystemFolderLog(
```

Comment:
rename to folder

Reply:
The method evaluates if the folder is the SYSTEM_FOLDER, that's why it has that name.

Reply:
done